mw-webapp/src/hooks/useClickOutside.ts

```diff
@@ -0,0 +1,26 @@
+import React, {useEffect} from "react";
+
+/**
+ *Custom hook to useClickOutside
+ */
+export const useClickOutside = (
+  refElem: React.RefObject<HTMLUListElement>,
+  callback: () => void,
+) => {
+
+  /**
+   *HandleClickOutside
+   */
+  const handleClickOutside = (e: Event) => {
+    const target = e.target as HTMLDivElement;
```

Comment:
Does it possible to avoid using "as" operator? Why should it be HTMLDivElement? If you will refactored dropdown component with radix, probably you can get rid of custom useClickOutside

Reply:
In this case, we can check if the target object is an instance of the HTMLElement:
const target = e.target;
if (target instanceof HTMLElement) {)
--
add rasix interface to component, delete custom useClickOutside